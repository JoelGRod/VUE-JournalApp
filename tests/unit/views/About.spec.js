import { shallowMount } from "@vue/test-utils";
import About from "@/views/About";

describe('Testing About.vue', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallowMount( About );
    });

    test('should match with snapshot', () => {
        // Arrange
        // Act
        // Assert
        expect( wrapper.html() ).toMatchSnapshot();
    });
})