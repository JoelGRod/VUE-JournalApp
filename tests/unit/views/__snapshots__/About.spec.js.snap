// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing About.vue should match with snapshot 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
